App/Logic/DataUploadLogic.swift

```diff
@@ -310,3 +320,32 @@ private extension CodableTemporaryExposureKey {
     )
   }
 }
+
+private extension UIApplication {
+  // The new UX in iOS 13.7 switches the whole UIWindow instead of just presenting an alert. This has the effect of
+  // breaking Tempura's internal navigation logic until the key window is restored.
+  // As a workaround, the app now waits for the key window to go back to its original instance
+  func waitForWindowRestored() -> Promise<Void> {
+    return Promise<Void> { resolve, reject, _ in
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/DataUploadLogic.swift;
  Line: 329;
-->

- :warning: swiftlint (0.39.2): Unused parameter "reject" in a closure should be replaced with _. (unused_closure_parameter)